### PR TITLE
Tabaxi Mercenaries (spellblades)

### PR DESCRIPTION
**Pull request by notafurryy**
## About The Pull Request

Long lost is your desert home, the warm sands, the clear skies.. None of that you can return to.. Forced to dwell in this swamp with your particular mastery of both blade and arcane making you a quite sought after mercenary, that is why your Holy Order sent you here, and other emissaries like you.

Adds a new type of mercenary using light armor and magic instead of the boring medium armor perk and sword/mace/polearms usage

Sadly don't have the skills to make custom sprites and custom magic weapons for now but that may change in the future!

I am open to feedback!

image

## Why It's Good For The Game

In a world with such magic potential surely someone must have thought to weaponize it on a larger scale. Makes sense to have in the world and adds some flavour to tabaxi while giving something unique to the mercenary role.

### CODE REVIEW

**Comment by Kevak** [comment]
If I'm counting right. That's.. Six spellpoints? Bit less than apprentice which makes sense considering it's a bladesinger adjacent. (Though do mind general comparisons to bladesinger's balance when considering skills for this one.)

Mayhaps consider permitting wildkin and halfkin. More kitty variety options would be nice rather than a pure mono-race thing. Alternatively give the grenzils a battlemage or something too.

**Comment by Gyrandola** [comment]
I am very scared of tabaxi with +3 speed and 5 athletics. Gonna need some balancejaks to chime in.

**Comment by Gyrandola** [comment]
Also, consider removing tabaxi from the Forlorn Hope now that they have their own mercenaries.

**Comment by Kevak** [comment]
> > If I'm counting right. That's.. Six spellpoints? Bit less than apprentice which makes sense considering it's a bladesinger adjacent. (Though do mind general comparisons to bladesinger's balance when considering skills for this one.)
> 
> It's 8 spell points since each level of arcane skill adds 1 and I more so balanced them around other mercenaries than Bladesingers that are supposed to be common wanderers. Spellblades are supposed to be the elite and they get little armor and stats compared to the others mercs with most light armors not being stab proof combined with the low constituation to really make them the glass cannons they are supposed to be.
> 
> And for the race thing I probably will consider adding other races but wild and half kin feels a little too broad to be on theme with the swift desert warriors, but maybe!

Light armor at start means next to nothing. Anybody can scrounge up enough for full plate with fairly little effort. Then you've got a gaggle of fairly potent mages in full plate with solid combat skills and batshit speed + legendary/near-legendary athletics. I would be extremely wary of adding a big speed buff to a species that already has very high speed, it'd be the tabaxi rogue problem electric boogaloo, this time with magic. Remember that speed plays a huge part in crit chance. I'd expect the meta with the current stat loadout to involve daggers and lightning bolt- Zap to knock prone, stab intent to bypass any armor and instantly slit throat.

Do consider enabling wildkin/halfkin. Although both wildkin/halfkin do tend to have a fairly wide species spread, the most common ones are either kitties or puppers. (Honestly I could make a solid argument for verminfolk too despite their shunned status in enigma considering one of the smallest kitties in the world are tiny lil desert kitties with an extremely high successful hunt rate, and being from a totally different part of the world, it is likely that the merc band would _not_ possess the same stigmas as the locals. On top of the whole 'tiny critters need less food/water' thing, which would have been important in the desert.)

**Comment by RadiantFlash7** [comment]
Righty. As the person who balanced the magic recently, and the one who last buffed bladesinger...I don't know how to feel about this. 

On one hand, this is literally blade singer, but better in almost every single way, and no longer rare (blade singers have a 25% chance to roll, like other rare classes for adventurers). The only difference between this and blade singer, is blade singer actually has quite a bit worse skills, a worse starting magic knowledge, and is based around upgrading their armor, rather then dodge expert like this PR.

-----------------

As for actual balance, this merc is rather wild. Athletics needs to drop to 3. Climbing should probably be 2 (mercs have little reason to climb). As for the stats, the stat total of +5 is actually significantly worse then all other mercs. Forlorn and grenzels have a +8 total bonus to stats. Sellswords have a +6, desert riders have a +8.

However... +3 speed is wild for a merc role. The highest speed roles currently are rogues and knaves, with I believe rangers as a second closest. I would suggest drop speed to +2 at a minimum, and give them an extra Perception, perhaps? I'd even be willing to say remove the con malus, because it's a merc role. having 6-8 stat point total /should/ be fine. 

When it comes to gear-Most of the mercs have a good starting piece or two of armor, and a steel weapon of their choice. Be it forlorn with helmet and chestpiece, Grenzel with meh armor all around and their fancy OP Chest plate, desert riders with scalemail and chain chausses, etc. This merc addition has hardened leather pants, a saber, and a haubergeon. Which- I suppose I don't hate. The hardened leather I squint at a bit, as its really only in the game to let people have the seamster upgrade lighter leather pants, rather then wearing chausses, like studded leather armor is upgraded by the smith. But mercs do start with good armor, so I suppose it's fine.

--------------------------------

Overall, this feels somewhat like you played the rare bladesinger role, and decided you wanted it more commonly. Which I dislike on principle.

However, I also do rather like the idea of a tabaxi mercs- EXCEPT: Desert riders are tabaxi-able mercs as well, and rather swift ones. So this doesn't really help much, tbqh.

**Comment by RadiantFlash7** [comment]
To add to this, I don't even think the role will be particularly effective, if I'm honest. Mages can't parry or dodge while casting spells. The lack of endurance stats, combined with tabaxi's natural lack of end means that you'll tire yourself out from dodging very, very fast, ontop of casting a spell.

**Comment by RadiantFlash7** [comment]
I'll throw up a buff to bladesingers I think. Given they're a rare roll, they've been a bit on the weaker side. Especially with something like this being merged.

**Comment by RadiantFlash7** [comment]
> adventurers dont need or deserve any buff

Rare roles are meant to be rare for a reason. If a reliably pickable is flat out better with no other benefits otherwise, It negates the point of having said rare role be rare.

File: code/modules/jobs/job_types/roguetown/mercenaries/types/newmoon.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/datum/advclass/newmoon
+	name = "New Moon Spellblade"
+	tutorial = "Knowledge is power.. You and your masters in the New Moon holy order took it to heart as you and all acolytes like you were forced to study countless text of both arcane and war to make yourself worthy a worthy emissary. Then it all crumbled, the news came while you were on your voyage; the bastion fell to the rot as only a small ammount of acolytes and spellblades escaped, most forced to retreat to this god forsaken swamp while others searched for something better."
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_races = list(/datum/species/tabaxi)
+	outfit = /datum/outfit/job/roguetown/mercenary/newmoon
+	maximum_possible_slots = 10
```
[the remark this answers is not in the document]
Comment (RadiantFlash7) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/newmoon.dm:7]: So-So honestly.  Casting spells means you can't dodge or parry.

File: code/modules/jobs/job_types/roguetown/mercenaries/types/newmoon.dm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		H.mind.adjust_spellpoints(4)
+		if(H.age == AGE_OLD)
+			head = /obj/item/clothing/head/roguetown/wizhat/gen
+			H.change_stat("intelligence", 4)
+			H.change_stat("strength", -2)
+			H.mind.adjust_skillrank(/datum/skill/magic/arcane, 1, TRUE)
+		H.change_stat("intelligence", 3)
+		H.change_stat("constitution", -2)
+		H.change_stat("endurance", 2)
+		H.change_stat("speed", 2)
```
[the remark this answers is not in the document]
Comment (RadiantFlash7) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/newmoon.dm:50]: Pretty much every other merc role has far far higher stats tbf. With the exception of sellsword

File: code/modules/clothing/rogueclothes/hats/hats.dm [hunk 3]
```diff
@@ -49,6 +49,10 @@
 	blocksound = SOFTHIT
 	sewrepair = TRUE
 
+/obj/item/clothing/head/roguetown/roguehood/reinforced/newmoon/Initialize()
+	color = pick(	color = pick(CLOTHING_WHITE, CLOTHING_BLUE))
```
Comment (Gyrandola) [inline code/modules/clothing/rogueclothes/hats/hats.dm:53]:
```suggestion
	color = pick(CLOTHING_WHITE, CLOTHING_BLUE)
```

**Comment by notafurryy** [comment]
image

New coat! Also changes the clothing to be teal

**Comment by Gyrandola** [comment]
image

fix the icon diff, you definitely did not do it right

**Comment by notafurryy** [comment]
> fix the icon diff, you definitely did not do it right

I have no clue how that happened but, fixed!

**Comment by RadiantFlash7** [comment]
> image
> 
> "New" coat (Lazy resprite of the old hand jacket)! Also changes the clothing to be teal Coat has about the same stats as the haubergeon so it souldn't impact balance much. Might also make a new sword for them!

I would consider re-adding their hardened leather pants tbh.
As it stands they only have a single good piece of armor (haubergeon), and all mercs have atleast 2.

**Comment by RadiantFlash7** [comment]
> These are so incredibly busted in combat.
> 
> Tabaxi-locked mages that get a decent chunk of STR (only -2), combat skills and spellpoints makes werewolves basically impossible to play, bandits can still frag them by abusing shields but this is NOT a good addition (especially considering the bonus END and SPD basically negates one of Tabaxi's main hardcounters, which is using moondust to outstam them!)
> 
> In practice, these have been nothing but a bane that exist to do nothing but run at an antagonist. I have not seen them really contribute anything that cannot and has not been contributed by the odd adventurer running about; except now these are a dedicated role that requires _no luck_ to get.
> 
> -SUPPORT, this should be a rare adventurer job, (it already is, its called bladesinger) instead of a multi-slot combat mage job.
> 
> If you want to see more combat magicians, buff bladesinger (so its worth taking) and make it less rare. Don't add a multi-slot job that can be stacked with multiple people.

It is absolutely not busted at all.
Tabaxi's with the +2 END bonus from this class often have at MOST, 11 endurance. Every single combat role, as more END and stamina. For that matter, Dodging also takes a fuckload of stamina more then parrying, which this class is geared towards.

Mind: Anyone using magic can't block, nor dodge while casting. Consider the alternative- You could be facing down a grenzel with expert polearms and an eaglesbeak. Frankly, This class is underpowered at present compared to other merc roles stats/skillwise, and it kind of needs to be that way to not outshine sellsword.  Nevermind the con penalty they have with this.

**Review by Gyrandola — APPROVED** [review]

**Pull request merged**